config.js

```diff
@@ -220,7 +220,7 @@ config.CHUNK_SPLIT_DELTA_CHUNK = config.CHUNK_SPLIT_AVG_CHUNK / 4;
 // CODER
 config.CHUNK_CODER_DIGEST_TYPE = 'sha384';
 config.CHUNK_CODER_FRAG_DIGEST_TYPE = 'sha1';
-config.CHUNK_CODER_COMPRESS_TYPE = 'snappy';
+config.CHUNK_CODER_COMPRESS_TYPE = process.env.DISABLE_COMPRESSION === 'true' ? undefined : 'snappy';
```

Comment:
Since this is a global env in the Pod, I would avoid setting such a generic env name that might affect other processes/libraries, and would prefix this with NOOBAA_
```suggestion
config.CHUNK_CODER_COMPRESS_TYPE = process.env.NOOBAA_DISABLE_COMPRESSION === 'true' ? undefined : 'snappy';
```

Reply:
Done